### ISSUE TITLE
An option to dump free non-levelcapped followers into missions awarding XP

### ISSUE THREAD

**Issue by FREEZEball** [teelolws/TLDRMissions#160]
TLDR version: 
An option to dump free non-levelcapped followers into missions awarding XP, possibly even ignoring success chance

I often use 1 200 - 1 500 XP missions rewarding resource container and other 900 - 1 000 XP missions as low priority (after useful things and bonus XP missions) means to level up. Basically, every follower not used in useful missions is dumped into those missions with no regard to success chance (I don't need resources from it as my banks are filled with them). "TLDR" is either not using those missions at all leaving a dozen of slacking bellow level 60 followers, or tries to maximize those missions, wasting anima, depending on "Crafting Materials" and other checkmarks in options. Is it possible to add an option to treat all unused missions as "leveling up" missions, ignore success chance on those and dump all free followers there, with priority from highest XP to lowest?

**Comment by teelolws** [comment]
There are other addons that do this already. Its outside the scope of my addon.
